crates/cargo-bsp/src/cargo_communication/cargo_types/cargo_command.rs

```diff
@@ -68,16 +79,11 @@ impl CreateCommand for RunParams {
         self.origin_id.clone()
     }
 
-    fn create_unit_graph_command(&self, root: &Path, targets_details: &[TargetDetails]) -> Command {
-        let targets_args = targets_details_to_args(targets_details);
-        cargo_command_with_unit_graph(CommandType::Run, root, targets_args)
+    fn command_type(&self) -> CommandType {
+        CommandType::Run
     }
-
-    fn create_requested_command(&self, root: &Path, targets_details: &[TargetDetails]) -> Command {
-        let target_args = targets_details_to_args(targets_details);
-        let mut cmd = create_requested_command(CommandType::Run, root, target_args);
-        cmd.args(self.arguments.clone());
-        cmd
+    fn arguments(&self) -> Vec<String> {
+        self.arguments.clone()
```

Comment:
Tutaj też brakuje nowej linii

Reply:
To jest to "requested changes"? XD

Reply:
Nie, chciałam mieć pewność, że żadna komenda się teraz nie wywala